types/redux-infinite-scroll/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "experimentalDecorators": true,
```

Comment:
Thank you for the feedback Andy! This is my first time submitting to DefinitelyTyped so I used other libraries to guide me. I will look closely on each one of your points.